pkg/collector/corechecks/cluster/orchestrator/transformers/k8s/pod.go

```diff
@@ -292,12 +292,18 @@ func getConditionMessage(p *corev1.Pod) string {
 
 // mapToTags converts a map for which both keys and values are strings to a
 // slice of strings containing those key-value pairs under the "key:value" form.
+// if the map contains empty values we only use the key instead
```

Comment:
I think there's a small issue with the history of the branch, it includes non-related commits #12858 in this case